bound/bound.go

```diff
@@ -4,7 +4,40 @@
 
 package bound
 
+import "math"
+
 // Bound represents [Min, Max] bounds.
 type Bound struct {
 	Min, Max float64
 }
+
+// IsEmpty returns whether the bound is empty
```

Comment:
Can we just call this IsNaN? If we have `IsNaN` and `IsValid` it's clear from just the names what they do. With `IsEmpty` and `IsValid` I can't tell the difference from the name alone.

@kortschak ?

Reply:
Nice. `IsNaN` SGTM.

Reply:
Do we need it though? If we have `IsValid`, is that not enough?

Reply:
IsValid is probably enough